docs/source/en/advanced/view-plugin.md

```diff
@@ -75,7 +87,7 @@ The three parameters of the `render` method are
 
 * filename: is the path to the complete file. The framework determines if the file exists when it looks for the file. It does not need to be processed here.
 * locals: The data needed for rendering. The data comes from `app.locals`, `ctx.locals` and calls `render` methods. The framework also has built-in `ctx`, `request`, `ctx.helper` objects.
```

Comment:
built-in 是形容词，这里的语义是“内置了” 为动词，故需要分开写

Reply:
I don't agree. You might be confused by thinking in Chinese. :smile: but I will do as you say since you have the authority here.

See this for example: https://www.treehugger.com/bikes/cyclevision-bike-helmet-has-built-cameras-front-and-rear.html